pandas/tests/groupby/test_grouping.py

```diff
@@ -547,7 +547,7 @@ def test_groupby_level_index_names(self, axis):
         df = DataFrame({"exp": ["A"] * 3 + ["B"] * 3, "var1": range(6)}).set_index(
             "exp"
         )
-        if axis in (1, "columns"):
+        if DataFrame()._get_axis_number(axis)==0:
```

Comment:
you can do this here, but use existing objects such as `df._get_axis_number(axis) == 1`. Also note you have got the 0 and 1 wrong here.

Reply:
Ah. Thanks for point that out, I'll correct it and re-commit